lib/queryBuilder/graphUpserter/UpsertNode.js

```diff
@@ -154,6 +128,10 @@ function hasOption(node, optName) {
   }
 }
 
+function decideType(node, defaultType, option, optionType = UpsertNodeType.None) {
```

Comment:
Node 4 and 5 don't support default values.

Reply:
Oh I keep forgetting that you still target these versions. Will adjust now.